catalog/app/containers/Admin/UsersAndRoles/RoleSelect.tsx

```diff
@@ -122,7 +124,14 @@ export function RoleSelect({
         </M.Typography>
       ) : (
         <M.Typography variant="body2" color="textSecondary">
-          User can assume any of the assigned roles
+          {nonAssignable ? (
+            <>
+              Roles assigned via role mapping and may be changed in config.
```

Comment:
```suggestion
              Roles are assigned via role mapping and may be changed in config.
```

@sir-sigurd @akarve better wording ideas?

Reply:
I'm not sure catalog can tell the reason with what's currently returned from GraphQL